TrkDiag/src/BkgDiag_module.cc

```diff
@@ -313,8 +303,16 @@ namespace mu2e
       double sqrSumDeltaTime(0.);
       double sqrSumDeltaX(0.);
       double sqrSumDeltaY(0.);
+      float zmin(0.);
+      float zmax(0.);
+      std::vector<float> hz;
+      std::vector<int> hp;
       for(auto const& ich : cluster.hits()){
         ComboHit const& ch = _chcol->at(ich);
+        if(ch.pos().Z() < zmin) zmin = ch.pos().Z();
```

Comment:
zmin and zmax seem irrelevant now you have a sorted vector to fill _zmin and _zmax